.goreleaser.yaml

```diff
@@ -0,0 +1,26 @@
+project_name: in-toto
+builds:
+  - ldflags:
+      - "-s -w"
+      - "-extldflags=-zrelro"
+      - "-extldflags=-znow"
+      - "-X main.tag={{.Version}}"
+      - "-X main.commit={{.FullCommit}}"
+      - "-X main.date={{.Date}}"
```

Comment:
ah, now I see why support for SDE is hard. I wonder if there's a way to use a heredoc or something like that ...

Reply:
I had a chat with the goreleaser maintainer.. he told me we should use CommitDate instead of Date. This way our build should be reproducible, because we would not use the date of the build environment. What do you think?

Reply:
I think this is a good idea!

Reply:
@SantiagoTorres Done.